index.js

```diff
@@ -540,4 +540,14 @@ DumbPipe.registerOpener("reload", function(message, sender) {
 
 DumbPipe.registerOpener("exit", function(message, sender) {
   window.close();
-});
\ No newline at end of file
+});
+
+DumbPipe.registerOpener("backgroundCheck", function(message, sender) {
+  if (navigator.mozAlarms) {
+    navigator.mozAlarms.add(new Date(Date.now() + 10000), 'ignoreTimezone', {});
+
+    navigator.mozSetMessageHandler('alarm', function() {
+      navigator.mozAlarms.add(new Date(Date.now() + 10000), 'ignoreTimezone', {});
```

Comment:
I picked 10 seconds for testing, I'm not sure what value would be good. Maybe 1 minute?